libompd/src/omp-icv.cpp

```diff
@@ -236,6 +238,38 @@ static ompd_rc_t ompd_get_debug(
   return ret;
 }
 
+static ompd_rc_t ompd_get_display_affinity(
+    ompd_address_space_handle_t *addr_handle, /* IN: handle for the address space */
+    ompd_word_t *display_affinity_val         /* OUT: display affinity value */
+    ) {
+  ompd_address_space_context_t *context = addr_handle->context;
+  if (!context)
+    return ompd_rc_stale_handle;
+  ompd_rc_t ret;
+
+  assert(callbacks && "Callback table not initialized!");
```

Comment:
As a general rule, putting asserts into library code is not good, because it will crash the tool. I think a better approach would be to return an error code to the tool. Calling assert() should be the last resort, not for recoverable error handling. I know that there are other places in the LLVM OMPD library that use assert(), but I think those should be fixed too.